src/az390.java

```diff
@@ -2410,6 +2421,41 @@ else if (tz390.op_trace_type[index]==613)                          // #533
 
 
 
+/**
+ * Comparator for mnemonics to mimic EBCDIC collating sequence
+ * The mnemonic portion occupies the first 8 positions of the ReportEntry
+ * and contains only uppercase characters and digits.
+ * Digits should sort higher than alphabetic characters.
+ */
+public class ReportEntryComparator implements Comparator<String>                   // #568
+ {@Override                                                                        // #568
+  public int compare(String str1, String str2)                                     // #568
+   {// special case: entries consisting of a single space sort first!              // #568
+    if (str1.charAt(0) == ' ') return 1;  // blank entry always smaller            // #568
+    if (str2.charAt(0) == ' ') return -1; // blank entry always smaller            // #568
```

Comment:
What if both strings are blank?

Reply:
Then they are the same and the sequence does not matter.

Reply:
If line 2434 is trun, code returns 1, not checking whether str2.charAt(0) is blank. Seems to me that code should return 0 in this case.

Reply:
As the comments indicate, a blank entry is by definition smaller. Even if they both are blank, equality does not matter, we simply declare the first blank we encounter the smaller of the two. Since the entries are discarded after sorting, the order of the blank entries is irrelevant, as long as they do sort up front.
I could add code to detect equality of entries staged for suppression, but that's more code and - to my mind - more confusing. If you still are opined differently, I'll make the change, though. As stated - it's irrelevant. What's more readable is a matter of style and taste :-)